darksocket_ros:
  topics:
    camera: /camera/rgb/image_color
    detections: /darksocket_ros/detections
  conf_path: /home/bwilab/scavenger_hunt/src/darksocket/scripts/darksocket.conf
  telemetry: false
